app/views/invoices/_invoice.html.haml

```diff
@@ -1,10 +1,11 @@
 %tr
-  %td= invoice.invoice_number
-  %td= invoice.client.name
-  %td= l invoice.paid_to_date if invoice.paid_to_date.present?
-  %td= l invoice.date_of_issue if invoice.date_of_issue.present?
-  %td= invoice_total_formatted(invoice)
-  %td= invoice.status.gsub('_',' ')
+  %td= index+1
+  %td.tbl-header-center= invoice.invoice_number
+  %td.tbl-header-center= invoice.client.name
+  %td.tbl-header-center= l invoice.paid_to_date if invoice.paid_to_date.present?
+  %td.tbl-header-center= l invoice.date_of_issue if invoice.date_of_issue.present?
+  %td.tbl-header-center= invoice_total_formatted(invoice)
+  %td.tbl-header-center= invoice.status.gsub('_',' ')
```

Comment:
Use rails' string conversion method instead of gsub.
Refer: http://apidock.com/rails/String/humanize)